api/database/seeders/CommunityRandomSeeder.php

```diff
@@ -15,6 +16,8 @@ class CommunityRandomSeeder extends Seeder
      */
     public function run()
     {
+        $faker = Faker::create();
+
```

Comment:
Does not appear to be used for anything

![image](https://github.com/user-attachments/assets/c95fb78b-fb10-458d-9df2-d958ff29b2d7)

If not needed, do delete the import too

Reply:
Nice catch, removed in [Remove unused variable](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13406/commits/58d90859bcae821d250bd0931490ffde8fe4b5e4).